aes-kw/src/error.rs

```diff
@@ -1,12 +1,20 @@
 use core::fmt;
 
+/// Result type with the `aes-kw` crate's [`Error`].
+pub type Result<T> = core::result::Result<T, Error>;
+
 /// Errors emitted from the wrap and unwrap operations.
 #[derive(Debug)]
 pub enum Error {
     /// Input data length invalid.
     InvalidDataLength,
     /// Invalid kek size.
     InvalidKekSize(usize),
+    /// Output buffer size invalid.
+    InvalidOutputSize {
+        /// Expected size in bytes.
+        expected: usize,
+    },
     /// Integrity check did not pass.
     IntegrityCheckFailed,
```

Comment:
Seems like these could be a little more consistent:

- `*Length` vs `*Size`
- Tuple variant vs one with named fields

Reply:
I'd probably suggest settling on `*Size` (it's what we tend to use in other crates, and also shorter).

I'd also suggest changing `InvalidKekSize { size: usize }` for consistency.

Reply:
yeah, makes sense to standardize on size